src/distributors/adoptopenjdk-installer.ts

```diff
@@ -0,0 +1,80 @@
+import * as core from '@actions/core';
+import * as tc from '@actions/tool-cache';
+
+import fs from 'fs';
+import path from 'path';
+import semver from 'semver';
+
+import { IS_WINDOWS, PLATFORM, IS_MACOS, macOSJavaContentDir } from "../util";
+import { JavaInstallerResults, JavaDownloadRelease, JavaBase, JavaInstallerOptions } from "./base-installer";
+import { IRelease, IReleaseVersion } from './adoptopenjdk-models'
```

Comment:
Different quotes across all imports and code, please, standardize them.

Reply:
Done. Thank you.